Makefile

```diff
@@ -55,7 +55,7 @@ fmt-check:
 		  exit 1 )
 
 fmt-fix:
-	sbt scalafmtAll scalafmtSbt
+	sbt "scalafix RemoveUnused" scalafmtAll scalafmtSbt
```

Comment:
did you consider running scalafix on test sources? `scalafixAll` can help with that (just like `scalafmtAll`)

Reply:
We should do that! I'll make this change. Thanks very much for taking a look here, @bjaglin! You're tips are very helpful :pray: